src/store/modules/product/mutations.ts

```diff
@@ -6,6 +6,13 @@ const mutations: MutationTree <ProductState> = {
   [types.PRODUCT_SEARCH_UPDATED] (state, payload) {
     state.products.list = payload.products;
     state.products.total = payload.totalProductsCount;
+  },
+  [types.PRODUCT_CACHED_UPDATED](state, payload) {
+    state.cached = payload.cached
+  },
+  [types.PRODUCT_LIST_UPDATED] (state, payload) {
```

Comment:
I think we can use the same mutation `PRODUCT_SEARCH_UPDATED` or rename that mutation if it's not used anywhere else.

Reply:
Removed PRODUCT_SEARCH_UPDATED mutation.